RPG Wizardry/Assets/Scripts/Entities/Enemies/AEnemy.cs

```diff
@@ -1,168 +1,168 @@
-﻿using nl.SWEG.RPGWizardry.Player;
```

Comment:
Yay for merge-bs..
(Please don't commit/push files if you did not edit them, to keep crap like this to a minumum 😄 )